src/multiplayer-servers/multiplayer-services/game-server-wrapper.md

```diff
@@ -0,0 +1,217 @@
+# Game Server Wrapper
+
+The game server wrapper, also known as the wrapper, gswrapper or gsw, can be used to run your game server executable
```

Comment:
```
The game server wrapper, also known as `gswrapper` or `gsw`, can be used to launch your game server executable inside the container image that runs your GameFabric Armadas or Vessels.

By starting the executable as a wrapper it can enable various convenience features.
```